Gem/Code/Source/Automation/ScriptReporter.cpp

```diff
@@ -15,6 +15,9 @@
 #include <AzFramework/IO/LocalFileIO.h>
 #include <AzCore/IO/SystemFile.h>
 #include <AzCore/Utils/Utils.h>
+#include <iostream>
```

Comment:
FYI `iostream` isn't used anywhere I don't believe and I think `AZ::IO::SystemFile` is the O3DE abstraction used for File I/O in general (you can look at `Serialization.cpp` for usage examples) 😊